soh/src/overlays/actors/ovl_En_Go2/z_en_go2.c

```diff
@@ -622,10 +622,6 @@ s16 EnGo2_GetStateGoronDmtBiggoron(GlobalContext* globalCtx, EnGo2* this) {
                 }
                 
                 if(gSaveContext.n64ddFlag) {
-                    if (INV_CONTENT(ITEM_CLAIM_CHECK) != ITEM_CLAIM_CHECK) {
-                        return 0;
-                    }
```

Comment:
so is the idea we just never hit this without the claim check now?

Reply:
That's right. If you don't have the claim check, the text box that gives it to you doesn't appear anymore, so we shouldn't hit this.